content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,17 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
-
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] with link:/doc/developer/plugin-development/incrementals/[incremental] builds is often a preferred technique as it benefits the user by:
+- providing additional information about the plugins being downloaded (such as, available updates or security warnings and many more), 
```

Comment:
What does this mean, and how does it relate to incrementals? The update center does not emit security warnings for incremental builds.
Nor is the plugin installation manager tool able to _add_ more information, rather it gathers what is already present, no?